framework/CodeInterfaces/RAVEN/RAVENInterface.py

```diff
@@ -162,21 +162,16 @@ def generateCommand(self,inputFiles,executable,clargs=None,fargs=None, preExec=N
     returnCommand = executeCommand, outputfile
     return returnCommand
 
-  def createNewInput(self,currentInputFiles,oriInputFiles,samplerType,**Kwargs):
+  def initialize(self, runInfo, oriInputFiles):
```

Comment:
Should we add this method into the CodeInterfaceBaseClass?

Reply:
added